sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/GridUtils.groovy

```diff
@@ -6,5 +6,6 @@ package life.qbic.portal.sampletracking.components
  * @since 1.0.0
  */
 class GridUtils {
-    final static int TITLE_COLUMN_WIDTH = 400
+    final static int MAX_CODE_COLUMN_WIDTH = 400
+    final static int MAX_STATUS_COLUMN_WIDTH = 200
```

Comment:
utility class alert :D What about `GridSettings` ?

Reply:
I agree. However, do we want to change it for this release or in the next since it was already approved?

Reply:
you know the answer probably :) its an easy fix and I think we can apply it now

Reply:
I think it would be best to refactor this in the next iteration into its own grid class since these values are pretty grid specific. I am not sure that we need them in a global context at all.